hail/python/hailtop/utils/utils.py

```diff
@@ -108,11 +108,13 @@ def digits_needed(i: int):
 
 
 def grouped(n, ls):
+    if n < 1:
+        raise ValueError ('invalid value for n')
```

Comment:
There shouldn't be a space between `ValueError` and `('invalid...`

Reply:
Fixed